### ISSUE TITLE
Add log level

### ISSUE THREAD

**Issue by iamplo** [movableink/backpack#1]
I find it useful to distinguish between info, warn and error. Default can be info. Something like

[Info] _Message_
[Error] _Some error message_
[Warn] _Some warning..._

**Comment by roger-rodriguez** [comment]
Good point. Personally, I would love to see all of this implemented. I had a quick conversation with @lee and maybe the logging functionality would be better in Cropduster CD.log()
